buildpipeline/Core-Setup-Windows-BT.json

```diff
@@ -60,7 +60,62 @@
       "enabled": true,
       "continueOnError": false,
       "alwaysRun": false,
-      "displayName": "Build",
+      "displayName": "Install Signing Plugin",
```

Comment:
Just updated this, Core-Setup-Windows-BT is now the x86/x64 job, Core-Setup-Windows-Arm-BT is the arm/arm64 job.